Lib/cmd.py

```diff
@@ -108,7 +108,13 @@ def cmdloop(self, intro=None):
                 import readline
                 self.old_completer = readline.get_completer()
                 readline.set_completer(self.complete)
-                readline.parse_and_bind(self.completekey+": complete")
+                if 'libedit' in getattr(readline, '__doc__', ''):
```

Comment:
I don't think we want the docstring to become API, but it's getting used in too many places now. Would you mind exposing a dedicated attribute, like `readline.backend`?
(Preferably in a separate PR, which doesn't need to block this one.)

Reply:
I can work on that, but yes in a separate PR because we want to replace all the usages as such in the PR as well.